packages/ui/src/helpers/authenticator/constants.ts

```diff
@@ -8,87 +8,76 @@ import { countryDialCodes } from '../../i18n';
 export const defaultFormFieldOptions: DefaultFormFieldOptions = {
   birthdate: {
     label: 'Birthdate',
-    labelHidden: true,
     placeholder: 'Birthdate',
     type: 'date',
     autocomplete: 'bday',
     isRequired: true,
   },
   confirmation_code: {
     label: 'Confirmation Code',
-    labelHidden: true,
     placeholder: 'Code',
     type: 'number',
     autocomplete: 'one-time-code',
     isRequired: true,
   },
   confirm_password: {
     label: 'Confirm Password',
-    labelHidden: true,
```

Comment:
Same question as above

Reply:
Here's what the image authenticator looks like for example: 
![image](https://user-images.githubusercontent.com/43682783/199602143-6b5328e1-c52e-4128-9b40-c14f6b3e3dc0.png)
 
Looking at this twice, I think this looks okay, but just wanted to confirm this looks good to other people too.

Reply:
Playing around with it more, this still looks much better than not having placeholder at all: 
![image](https://user-images.githubusercontent.com/43682783/199603171-69c92107-69ad-4885-84bd-7948799ea251.png)

so scratch what I said above, I no longer feel this is a blocker 🙏

Reply:
I like the no placeholders. I wouldn't block on this, but ideally no placeholders